cmux-tui/scripts/verify_published_manifest.py

```
@@ -0,0 +1,178 @@
+            raise ManifestError(f"{source} missing required artifact {artifact}")
+
+
+def _read_url(url: str) -> bytes:
+    request = urllib.request.Request(
+        url,
+        headers={
+            "Accept": "application/json",
+            "Cache-Control": "no-cache",
+            "User-Agent": "cmux-tui-manifest-verifier/1 (https://github.com/manaflow-ai/cmux)",
+        },
+    )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)